Source/ACE.Server/start_server.bat

```diff
@@ -1 +1,3 @@
-dotnet ACE.Server.dll
\ No newline at end of file
+@cd %~dp0
+@if not exist "ACE.Server.dll" (if exist ".\bin\x64\Debug\netcoreapp2.1\ACE.Server.dll" (@cd ".\bin\x64\Debug\netcoreapp2.1\"))
```

Comment:
What about release build?

This assumes too much I think. These issues should also be reduced once we publish binaries.

Reply:
this was fixed in the configuration file format change PR